apps/web/src/components/(dashboard)/common/command-menu.tsx

```diff
@@ -85,7 +85,8 @@ export function CommandMenu({ open, onOpenChange }: CommandMenuProps) {
 
   const currentPage = pages[pages.length - 1];
 
-  const toggleOpen = () => {
+  const toggleOpen = (e: KeyboardEvent) => {
+    e.preventDefault();
```

Comment:
Makes sense, does `stopPropogation` or `stopImmediateProprogation` also work?

Reply:
No, it is not effective in this case because we are attempting to prevent the browser's default behaviour.

Reply:
Sweet, lets roll with this then!